oshmem/shmem/c/shmem_clear_lock.c

```diff
@@ -27,5 +28,10 @@
 
 void shmem_clear_lock(volatile long *lock)
 {
-    _shmem_clear_lock((void *)lock, sizeof(long));
+	if (oshmem_shmem_mcs_locks) {
+		SHMEM_API_VERBOSE(10, "shmem_clear_lock using MCS Lock");
```

Comment:
maybe move this print to the very beggining of the func and also print locking scheme being cleaned?

Reply:
I have moved the print and also printing the locking scheme now with the latest commit